### ISSUE TITLE
pgmoon openssl.rand not found

### ISSUE THREAD

**Issue by rs-joe** [Kong/kong#8533]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Kong version (`$ kong version`)

2.8.0

### Current Behavior

As I started the Install Kong Gateway on macOS guide, I attempted to execute the `kong migrations bootstrap` and encountered the error `/opt/homebrew/share/lua/5.1/pgmoon/init.lua:273: module 'openssl.rand' not found`.

I noticed there was a previous issue reported and has since been closed with release 2.8.0. https://github.com/Kong/kong/issues/8259

I expected that Kong v2.8.0 resolved this issue with updates associated with pgmoon dependency.

### Expected Behavior

`kong migrations bootstrap` completes without error.

### Steps To Reproduce

Have PostgreSQL 14 installed and running
Create Kong user and Kong Database
Install via Homebrew
Execute `kong migrations bootstrap`
[Optional] Create kong.conf and adjust the Postgres DB values, such as password and copy it to /etc/kong/kong.conf

### Anything else?

https://docs.konghq.com/gateway/2.8.x/install-and-run/macos/

**Comment by bungle** [comment]
@rs-joe, yes it was decided in a last minute to remove this from release. I tried to oppose the removal, but it was too late. The decision had to do with the issues that were caused to our sibling commercial release, and we didn't have time to resolve it properly. Thus we postponed the fix to next release, hopefully we can release a patch release soon.

The `master` branch still contains the fix:
https://github.com/Kong/kong/pull/8429

Basically you can just use `luarocks` to install latest `pgmoon` and you should be fine, but I agree it is annoying manual process until we get out packages fixed.

**Comment by rs-joe** [comment]
@bungle, I've used `luarocks` v3.5.0 (packaged with `kong` v2.8.0) to install `pgmoon` (`luarocks install pgmoon`). The install succeeded, but did not resolve the issue with the command `kong migration bootstrap`. Is there another step that I should complete in addition to the `pgmooon` install?

**Comment by bungle** [comment]
@rs-joe,

This is the line in `1.14.0`:
https://github.com/leafo/pgmoon/blob/v1.14.0/pgmoon/init.lua#L273

I feel like you are still hitting to:
https://github.com/leafo/pgmoon/blob/v1.13.0/pgmoon/init.lua#L273

You may have two versions installed. Try removing `pgmoon` with `luarocks remove --force pgmoon ` and then `luarocks install --force pgmoon 1.14.0`. Are you running the Open Source Kong or the Enterprise edition?

**Comment by rs-joe** [comment]
@bungle, I am currently working to develop a POC and utilizing the Open Source Kong. I did found a work-around until this can resolved in a future release. The issue is, as you stated, `pgmoon` `1.14.0` needs to be used.

In my case, I am using the Homebrew Kong formulae. So, the install of `pgmoon` `1.14.0` using command `luarocks install pgmoon` didn't have any immediate affect on Kong. Both Homebrew and Kong have a reference to `pgmoon` `1.13.0` buried in folders `/opt/homebrew/Cellar/kong/2.8.0/share/lua/5.1/pgmoon` and `/opt/homebrew/share/lua/5.1/pgmoon`.

In order to force the reference from `1.13.0` to `1.14.0`, I copied files from the `pgmoon` `1.14.0` using `luarocks` located here: `/opt/homebrew/Cellar/openresty@1.19.3.2/1.19.3.2/luarocks/share/lua/5.1/pgmoon` to both the directories referencing the old version (stated above). This step essentially overwrote the old `pgmoon` files with the new.

I figure if I upgrade Kong using Homebrew it will overwrite these files. So, I'll keep them as they are for now until the resolution is released.

Also, I found that if I only copied the files newer files in the Kong shared directory path I passed the original issue, but encountered a new one (https://github.com/Kong/kong/issues/4933). So, this is why it was necessary to overwrite the files in both the Homebrew and Kong shared directories.
